janus_core/calculations/md.py

```diff
@@ -423,14 +423,20 @@ def __init__(
                 "heating to run",
                 stacklevel=2,
             )
-        if self.ramp_temp and self.ensemble in ("nve", "nph"):
-            raise ValueError(
-                "Temperature ramp requested for ensemble with no thermostat."
-            )
+        if self.ramp_temp:
+            if self.ensemble in ("nve", "nph"):
+                raise ValueError(
+                    "Temperature ramp requested for ensemble with no thermostat"
+                )
 
-        # Check validate start and end temperatures
-        if self.ramp_temp and (self.temp_start < 0 or self.temp_end < 0):
-            raise ValueError("Start and end temperatures must be positive")
+            # Check validate start and end temperatures
```

Comment:
```suggestion
            # Validate start and end temperatures
```
Not your typo but while we're at it